daemon/config.go

```diff
@@ -73,6 +73,9 @@ type Config struct {
 	// values: { auto | always | policy }
 	AllowLocalhost string
 
+	// AutoV6L3Routes automatically adds L3 direct routing when using direct mode (-d)
+	AutoV6L3Routes bool
```

Comment:
1. Done

2. At least it would be easier to have a dedicated place with a structure to store the daemon's configuration. We could have that configuration on a different package to avoid import cycles, this way we could access the configuration from everywhere on the code. Then, by having a dedicated configuration structure we can have mutexes for flags that can be changed at runtime and hide the lock mechanism from the function caller.

Reply:
OK. I'm not against having a structure but there is no point in storing it inside `Daemon`. It only requires us to pass around this Daemon structure everywhere.

Reply:
Done!